webroot/results/includes/_database.php

```diff
@@ -189,11 +189,6 @@ function getAllIDs ( $rows ) {
 }
 
 function getAllEventIds                    () { return getAllIDs( getAllEvents()                    ); }
-function getAllRoundIds                    () { return getAllIDs( getAllRounds()                    ); }
-function getAllCompetitionIds              () { return getAllIDs( getAllCompetitions()              ); }
-function getAllUsedCountriesIds            () { return getAllIDs( getAllUsedCountries()             ); }
-function getAllUsedCountriesCompetitionIds () { return getAllIDs( getAllUsedCountriesCompetitions() ); }
-function getAllUsedContinentIds            () { return getAllIDs( getAllUsedContinents()            ); }
```

Comment:
Why are you specifically deleting these but not `getAllEventIds`?
I get that some of the methods here may be unused but we're not maintaining the PHP code anymore, so it's not worth making such fine-grained changes.

I'd rather leave these methods in an unused state, instead of accidentally deleting something you (or your IDE) _think_ is unused but actually gets invoked in an obscure, dynamic way in some legacy results admin code (which then could break)

Reply:
`getAllEventIds` is/was used here https://github.com/thewca/worldcubeassociation.org/blob/bdc4d8cb8da360e9d6e669cd48b94e447cb29c02/webroot/results/admin/migrate/migration9.php#L81, that's why I kept it. Anyways, I put everything back after your review =]